src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -183,6 +195,10 @@ public BoundType GetBoundAttributeType(AttributeData attribute)
 			{
 				var isDbPropertyAttributeArgs = attribute.NamedArguments.Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase)).ToList();    //case insensitive check
 
+				if (isDbPropertyAttributeArgs.Count == 0 && // IsDBField property defined in base class 
+					TypesContainsIsDBField.Any(typesContainsIsDBField => IsAttributeDerivedFromClass(attribute.AttributeClass, typesContainsIsDBField))) 
```

Comment:
Can you please move linq query to a local variable to make the if condition more readable